src/index.js

```diff
@@ -1,43 +1,35 @@
-import AnchorSource from './components/Anchor'
-import AvatarSource from './components/Avatar'
 import ButtonSource from './components/Button'
-import CardSource from './components/Card'
 // import CategoriesBrowserImport from './components/CategoriesBrowser'
-// import FooterImport from './components/Footer'
-import CourseCardSource from './components/Card/CourseCard'
-import ErrorSource from './components/Error'
-import LessonCardSource from './components/Card/LessonCard'
-import PlaylistCardSource from './components/Card/PlaylistCard'
 import CheckboxSource from './components/Checkbox'
+import CourseCardSource from './components/CourseCard'
```

Comment:
@tayiorbeii That's the beauty of the `src/index.js` API. Since I didn't change the names of the exported modules, just the paths - it is an implementation detail that shouldn't effect them. As long as they aren't using `Card` directly, which I removed from the exports. Does that make sense?